homeassistant/components/matter/strings.json

```
@@ -307,6 +307,16 @@
       },
       "water_heater_mode": {
         "name": "Water heater mode"
+      },
+      "door_lock_operating_mode": {
+        "name": "Operating Mode",
```

Due to [HA naming conventions](https://developers.home-assistant.io/docs/core/entity/#entity-naming:~:text=Entity%20names%20should%20start%20with%20a%20capital%20letter%2C%20the%20rest%20of%20the%20words%20are%20lower%20case%20(unless%20it%27s%20a%20proper%20noun%20or%20a%20capitalized%20abbreviation%20of%20course).), this should be "Operating mode":

```suggestion
        "name": "Operating mode",
```